src/RPCAL/nfs_dupreq.c

```diff
@@ -466,7 +466,7 @@ static inline void drc_free_expired(void)
 	DRC_ST_LOCK();
 
 	if ((drc_st->tcp_drc_recycle_qlen < 1) ||
-	    (now - drc_st->last_expire_check) < 600) /* 10m */
+	    (now - drc_st->last_expire_check) < drc_st->expire_delta)
```

Comment:
The purpose of expire_delta seems to be different. If it's set to a small value we do not want to run the below algorithm so frequently.